cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/parameters/ParameterUtils.java

```diff
@@ -467,6 +469,11 @@ static long time(HttpServletRequest request) {
       return System.currentTimeMillis();
     }
 
+    if (startMs(request) != null || endMs(request) != null) {
```

Comment:
I wonder whether we can relax this sanity check a little. Are there any side effects of allowing the use of `time` together with `startMs`?

Reply:
No, I think it is a little counter-intuitive that `time`  refers to end time, but this seems the debt we have to pay to maintain back-compatibility.